docs/source/workflow_outputs.rst

```diff
@@ -460,3 +461,31 @@ Design loads are based on block load ACCA Manual J calculations.
    Design Loads Cooling Latent: Infiltration/Ventilation (Btu/h)     Latent cooling design load for infiltration/ventilation
    Design Loads Cooling Latent: Internal Gains (Btu/h)               Latent cooling design load for internal gains
    ================================================================  ====================
+
+.. _bill_outputs:
+
+Utility Bill Outputs
+--------------------
+
+OpenStudio-HPXML can optionally generate a utility bills output file.
+The utility bills output file is called ``results_bills.csv`` (or ``results_bills.json``) and located in the run directory.
+Monthly fixed charges and marginal rates, as well as PV compensation types/rates/fees, are defined to determine how utility bills are calculated.
+
+The file includes:
+
+   =============================================  ====================
+   Type                                           Notes
+   =============================================  ====================
+   Electricity: Fixed ($)                         Annual fixed charges for electricity.
+   Electricity: Marginal ($)                      Annual energy charges for electricity.
+   Electricity: PV Credit ($)                     Annual production credit for PV.
+   Electricity: Total ($)                         Annual total charges for electricity.
+   Natural Gas: Fixed ($)                         Annual fixed charges for natural gas.
+   Natural Gas: Marginal ($)                      Annual energy charges for natural gas.
+   Natural Gas: Total ($)                         Annual total charges for natural gas.
+   Fuel Oil: Total ($)                            Annual total charges for fuel oil.
+   Propane: Total ($)                             Annual total charges for propane.
+   Wood Cord: Total ($)                           Annual total charges for wood cord.
+   Wood Pellets: Total ($)                        Annual total charges for wood pellets.
+   Coal: Total ($)                                Annual total charges for coal.
```

Comment:
@shorowit Should we populate these in `results_bills.csv` even when they are zero (i.e., when fuel type is not applicable)?

Reply:
Yes, let's populate them all regardless of fuel types in the building. That is most consistent with energy outputs (e.g., results_annual.csv) and eliminates surprises for users.